lib/rspec/mocks/proxy.rb

```diff
@@ -142,6 +145,11 @@ def received_message?(method_name, *args, &block)
       end
 
       # @private
+      def messages_arg_list
```

Comment:
`messages_arg_list` was the first name that came to mind, but I don't really like it.  Should I leave it, or perhaps change to `message_args`, `args_from_messages`, or perhaps just swap `for` and `from` to make `args_from_multiple_calls`. Might be too confusing with the name being so close to `args_for_multiple_calls`.